meerkat_abacus/orchestrate.py

```diff
@@ -45,7 +45,7 @@ def on_configure(self):
 global engine
 global session
 
-tz = pytz.timezone(config.timezone)
+tz = pytz.timezone(config.country_config.get("timezone", "Europe/Dublin"))
```

Comment:
I don't like the fact that we have the default value configured in many places.
`Europe/Dublin` is here and in 
```python
# meerkat_abacus/util/__init__.py
def send_alert(alert_id, alert, variables, locations, param_config=config):
[...]
local_timezone = pytz.timezone(param_config.country_config.get(
                    "timezone", "Europe/Dublin"
                ))
```